infra.bs

```diff
@@ -280,6 +280,81 @@ quotes and monospace font.
 <a>ASCII lowercase</a> of <var>B</var>.
 <!-- TODO: define string equals? -->
 
+To <dfn data-export="">collect a sequence of characters</dfn> means that the following algorithm
```

Comment:
Missing a `<p>`.

I also think we want to rephrase this to make the arguments explicit.

(The other thing I'm a little unsure about is "characters", since that's not a term we currently use, but perhaps it's best for now since we haven't really figured out how we want to do all aspects of strings yet.)